src/DotQuery.Core/Caches/EmptyQueryCache.cs

```diff
@@ -45,5 +47,9 @@ public TValue GetOrAdd(TKey key, TValue lazyTask, EntryOptions options)
         public void Set(TKey key, TValue value, EntryOptions options)
         {
         }
+
+        public void Remove(TKey key)
```

Comment:
Mee too. However the `Microsoft.Extensions.Caching.Memory.IMemoryCache` just has a method with the signature `void Remove(object key)` and doesn't return `boolean`.